### DISCUSSION TITLE
When 1.5 Release?

### DISCUSSION THREAD

**Discussion by FlorianNoeverGOB** [rvanbekkum/vsc-xliff-sync#124] [General]
Hello,
just wanted to ask when you roughly plan to release the (I guess 1.5) update to my [PR](https://github.com/rvanbekkum/vsc-xliff-sync/pull/122) 😊

**Comment by rvanbekkum** [comment]
Hi @FlorianNoeverGOB,
I was/am on a holiday/vacation, so I will most likely release it at the start of next week. 😊

**Comment by rvanbekkum** [comment]
Yesterday, the 1.5.0 release was created. See also: https://github.com/rvanbekkum/vsc-xliff-sync/discussions/125

Thanks again for your contributions. 😊
